SCIDB QUERY : <create array SCHEMA <v:int64>[i=0:7;j=0:7]>
Query was executed successfully

SCIDB QUERY : <store(redimension(apply(build(<v:int64>[i=0:7], i),j,i), SCHEMA), DATA)>
Query was executed successfully

SCIDB QUERY : <store(flatten(build(DATA, i)), DF)>
Query was executed successfully

SCIDB QUERY : <store(DATA, DATA_EXISTS)>
Query was executed successfully

SCIDB QUERY : <store(DF, DF_EXISTS)>
Query was executed successfully

SCIDB QUERY : <store(DATA, OUT_HA, distribution: hashed)>
Query was executed successfully

SCIDB QUERY : <show(OUT_HA)>
{i} schema,distribution
{0} 'OUT_HA<v:int64> [i=0:7:0:8; j=0:7:0:8]','hashed'

SCIDB QUERY : <scan(OUT_HA)>
{i,j} v
{0,0} 0
{1,1} 1
{2,2} 2
{3,3} 3
{4,4} 4
{5,5} 5
{6,6} 6
{7,7} 7

SCIDB QUERY : <store(DATA, OUT_RC, distribution: row_cyclic)>
Query was executed successfully

SCIDB QUERY : <show(OUT_RC)>
{i} schema,distribution
{0} 'OUT_RC<v:int64> [i=0:7:0:8; j=0:7:0:8]','row_cyclic'

SCIDB QUERY : <scan(OUT_RC)>
{i,j} v
{0,0} 0
{1,1} 1
{2,2} 2
{3,3} 3
{4,4} 4
{5,5} 5
{6,6} 6
{7,7} 7

SCIDB QUERY : <store(DATA, OUT_RE, distribution: replicated)>
Query was executed successfully

SCIDB QUERY : <show(OUT_RE)>
{i} schema,distribution
{0} 'OUT_RE<v:int64> [i=0:7:0:8; j=0:7:0:8]','replicated'

SCIDB QUERY : <scan(OUT_RE)>
{i,j} v
{0,0} 0
{1,1} 1
{2,2} 2
{3,3} 3
{4,4} 4
{5,5} 5
{6,6} 6
{7,7} 7

SCIDB QUERY : <store(DATA, OUT_CC, distribution: col_cyclic)>
Query was executed successfully

SCIDB QUERY : <show(OUT_CC)>
{i} schema,distribution
{0} 'OUT_CC<v:int64> [i=0:7:0:8; j=0:7:0:8]','col_cyclic'

SCIDB QUERY : <scan(OUT_CC)>
{i,j} v
{0,0} 0
{1,1} 1
{2,2} 2
{3,3} 3
{4,4} 4
{5,5} 5
{6,6} 6
{7,7} 7

SCIDB QUERY : <store(DF, NOT_EXISTS, distribution: hashed)>
[An error expected at this place for the query "store(DF, NOT_EXISTS, distribution: hashed)". And it failed with error code = scidb::SCIDB_SE_INFER_SCHEMA::SCIDB_LE_INVALID_OPERATOR_ARGUMENT. Expected error code = scidb::SCIDB_SE_INFER_SCHEMA::SCIDB_LE_INVALID_OPERATOR_ARGUMENT.]

SCIDB QUERY : <store(DF, NOT_EXISTS, distribution: row_cyclic)>
[An error expected at this place for the query "store(DF, NOT_EXISTS, distribution: row_cyclic)". And it failed with error code = scidb::SCIDB_SE_INFER_SCHEMA::SCIDB_LE_INVALID_OPERATOR_ARGUMENT. Expected error code = scidb::SCIDB_SE_INFER_SCHEMA::SCIDB_LE_INVALID_OPERATOR_ARGUMENT.]

SCIDB QUERY : <store(DF, NOT_EXISTS, distribution: replicated)>
[An error expected at this place for the query "store(DF, NOT_EXISTS, distribution: replicated)". And it failed with error code = scidb::SCIDB_SE_INFER_SCHEMA::SCIDB_LE_INVALID_OPERATOR_ARGUMENT. Expected error code = scidb::SCIDB_SE_INFER_SCHEMA::SCIDB_LE_INVALID_OPERATOR_ARGUMENT.]

SCIDB QUERY : <store(DF, NOT_EXISTS, distribution: col_cyclic)>
[An error expected at this place for the query "store(DF, NOT_EXISTS, distribution: col_cyclic)". And it failed with error code = scidb::SCIDB_SE_INFER_SCHEMA::SCIDB_LE_INVALID_OPERATOR_ARGUMENT. Expected error code = scidb::SCIDB_SE_INFER_SCHEMA::SCIDB_LE_INVALID_OPERATOR_ARGUMENT.]

SCIDB QUERY : <store(DF, DF_EXISTS, distribution: hashed)>
[An error expected at this place for the query "store(DF, DF_EXISTS, distribution: hashed)". And it failed.]

SCIDB QUERY : <store(DF, DF_EXISTS, distribution: row_cyclic)>
[An error expected at this place for the query "store(DF, DF_EXISTS, distribution: row_cyclic)". And it failed.]

SCIDB QUERY : <store(DF, DF_EXISTS, distribution: replicated)>
[An error expected at this place for the query "store(DF, DF_EXISTS, distribution: replicated)". And it failed.]

SCIDB QUERY : <store(DF, DF_EXISTS, distribution: col_cyclic)>
[An error expected at this place for the query "store(DF, DF_EXISTS, distribution: col_cyclic)". And it failed.]

SCIDB QUERY : <store(DF, DATA_EXISTS, distribution: hashed)>
[An error expected at this place for the query "store(DF, DATA_EXISTS, distribution: hashed)". And it failed.]

SCIDB QUERY : <store(DF, DATA_EXISTS, distribution: row_cyclic)>
[An error expected at this place for the query "store(DF, DATA_EXISTS, distribution: row_cyclic)". And it failed.]

SCIDB QUERY : <store(DF, DATA_EXISTS, distribution: replicated)>
[An error expected at this place for the query "store(DF, DATA_EXISTS, distribution: replicated)". And it failed.]

SCIDB QUERY : <store(DF, DATA_EXISTS, distribution: col_cyclic)>
[An error expected at this place for the query "store(DF, DATA_EXISTS, distribution: col_cyclic)". And it failed.]

SCIDB QUERY : <remove(SCHEMA)>
Query was executed successfully

SCIDB QUERY : <remove(DATA)>
Query was executed successfully

SCIDB QUERY : <remove(DF)>
Query was executed successfully

SCIDB QUERY : <remove(DATA_EXISTS)>
Query was executed successfully

SCIDB QUERY : <remove(DF_EXISTS)>
Query was executed successfully

SCIDB QUERY : <remove(OUT_HA)>
Query was executed successfully

SCIDB QUERY : <remove(OUT_RC)>
Query was executed successfully

SCIDB QUERY : <remove(OUT_RE)>
Query was executed successfully

SCIDB QUERY : <remove(OUT_CC)>
Query was executed successfully

